src/cm.rs

```diff
@@ -377,6 +377,7 @@ impl<'a, 'o> CommonMarkFormatter<'a, 'o> {
             NodeValue::FootnoteReference(ref nfr) => {
                 self.format_footnote_reference(nfr.name.as_bytes(), entering)
             }
+            NodeValue::MultilineBlockQuote(..) => self.format_block_quote(entering),
```

Comment:
For now just outputting a regular block quote, as opposed to outputting the actual multiline format. I _think_ this makes sense, though I could see both ways.

Reply:
Yes, this seems fine! If you want to be able to round-trip it, we can always add it back in later.